modules/http/src/main/java/org/glassfish/grizzly/http/HttpCodecFilter.java

```diff
@@ -627,7 +631,19 @@ public final NextAction handleRead(final FilterChainContext ctx,
                 LOGGER.log(Level.FINE, "Error parsing HTTP header", e);
 
                 HttpProbeNotifier.notifyProbesError(this, connection, httpHeader, e);
```

Comment:
Now this method won't receive the same exception than onHttpHeaderError, is it ok?